requirements.txt

```
@@ -43,3 +43,6 @@ XlsxWriter==0.9.3
 xlwt==1.3.*
 xlrd==1.0.0
 pypiwin32 ; sys_platform == 'win32'
+# Needed for OpenUpgrade
+odoorpc==0.7.0
+openupgradelib>=2.0.0
```

Please use for avoiding future problems
```suggestion
git+https://github.com/OCA/openupgradelib.git@master
```